### DISCUSSION TITLE
what needs to be substituted for '<PATH_TO_SYS>' in the command on Pg 17 chipsec manual v1.8.7

### DISCUSSION THREAD

**Discussion by girishd22** [chipsec/chipsec#1566] [Q&A] [answered]
Hi All,

Windows Installation section(Pg 17 of 99) of chipsec manual v1.8.7 has a command which creates chipsec entry in registry, See command below:

Command:
sc create chipsec binpath="<PATH_TO_SYS>" type= kernel DisplayName="Chipsec driver"

In the above command, what needs to be substituted for '<PATH_TO_SYS>' ?

Thanks,
Girish

**Comment by npmitche** [comment, marked as answer]
For windows, the driver (.sys file) should be located: `chipsec\helper\win\win7_x86\chipsec_hlpr.sys` or `chipsec\helper\win\win7_amd64\chipsec_hlpr.sys` depending on architecture.  

I believe it needs to be an absolute path, so be sure to include the path to where the chipsec folder is as well.

**Reply by girishd22** [reply]
Thank you so much Nathaniel Mitchell for your timely response! Including the path mentioned by you worked. Appreciate the help. 

Thanks,
Girish

**Reply by girishd22** [reply]
Hi Nathaniel Mitchell, 
I am observing an issue running fuzzmore test with Chipsec v1.9.1. Can you please respond to the Q & A #1603?

Thanks,
Girish
